rest/service/src/v0/nexuses.rs

```diff
@@ -1,75 +1,76 @@
 use super::*;
 
-struct Factory {}
-impl HttpServiceFactory for Factory {
-    fn register(self, config: &mut AppService) {
-        get_nexuses.register(config);
-        get_nexus.register(config);
-        get_node_nexuses.register(config);
-        get_node_nexus.register(config);
-        put_node_nexus.register(config);
-        del_node_nexus.register(config);
-        del_nexus.register(config);
-        put_node_nexus_share.register(config);
-        del_node_nexus_share.register(config);
-    }
-}
-pub(crate) fn factory() -> impl HttpServiceFactory {
-    Factory {}
+pub(crate) fn configure(cfg: &mut paperclip::actix::web::ServiceConfig) {
+    cfg.service(get_nexuses)
+        .service(get_nexus)
+        .service(get_node_nexuses)
+        .service(get_node_nexus)
+        .service(put_node_nexus)
+        .service(del_node_nexus)
+        .service(del_nexus)
+        .service(put_node_nexus_share)
+        .service(del_node_nexus_share);
 }
 
-#[get("/v0/nexuses")]
-async fn get_nexuses() -> impl Responder {
+#[get("/v0/nexuses", tags(Nexuses))]
+async fn get_nexuses() -> Result<Json<Vec<Nexus>>, RestError> {
     RestRespond::result(MessageBus::get_nexuses(Filter::None).await)
 }
-#[get("/v0/nexuses/{nexus_id}")]
-async fn get_nexus(web::Path(nexus_id): web::Path<NexusId>) -> impl Responder {
-    RestRespond::result(MessageBus::get_nexuses(Filter::Nexus(nexus_id)).await)
+#[get("/v0/nexuses/{nexus_id}", tags(Nexuses))]
```

Comment:
Nit: You didn't run rustfmt yet, there's no newline here.

Reply:
new line where? I just ran fmt again, seems happy

Reply:
Ah I think I'm just going mad. >.< Sorry!